docker-compose.yml

```
@@ -95,8 +95,11 @@ services:
       - frontend
     working_dir: /src
     image: node:20.13
-    command: >
-      /bin/sh -c 'yarn install && yarn storybook & yarn workspace mit-open watch:docker'
+    entrypoint: ["/bin/sh", "-c"]
+    command:
+      - |
+        yarn install --immutable
+        yarn workspace mit-open storybook & yarn workspace mit-open watch:docker
```

I'd suggest splitting this to another line too so it's more obvious you're sending storybook to the background as opposed to `&&` - it looked like the latter at a glance having them on one line.
```suggestion
        yarn workspace mit-open storybook &
        yarn workspace mit-open watch:docker
```